tsdb/ooo_head_read_test.go

```diff
@@ -468,19 +477,26 @@ func TestOOOHeadChunkReader_LabelValues(t *testing.T) {
 // It does so by appending out of order samples to the db and then initializing
 // an OOOHeadChunkReader to read chunks from it.
 func TestOOOHeadChunkReader_Chunk(t *testing.T) {
+	for name, scenario := range sampleTypeScenarios {
+		t.Run(name, func(t *testing.T) {
+			testOOOHeadChunkReader_Chunk(t, scenario)
+		})
+	}
+}
+
+// TestOOOHeadChunkReader_Chunk tests that the Chunk method works as expected.
+// It does so by appending out of order samples to the db and then initializing
+// an OOOHeadChunkReader to read chunks from it.
+//
```

Comment:
I think we can remove this comment because it's already commented in TestOOOHeadChunkReader_Chunk